contracts/middle-layer/resource-mirror/CmnHub.sol

```diff
@@ -68,6 +69,10 @@ abstract contract CmnHub is CmnStorage, Initializable, ICmnHub, IMiddleLayer {
     function updateParam(string calldata key, bytes calldata value) external virtual onlyGov {
         if (_compareStrings(key, "BaseURI")) {
             IERC721NonTransferable(ERC721Token).setBaseURI(string(value));
+        } else if (_compareStrings(key, "AdditionalContract")) {
+            require(value.length == 20, "length of additional address mismatch");
+            address newAdditional = BytesToTypes.bytesToAddress(20, value);
```

Comment:
it is better to make sure that `newAdditional` is contract address

Reply:
Done